app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/tei_data/TEIDataPresenterImpl.java

```diff
@@ -256,16 +251,7 @@ public void onShareClick(View mView) {
 
     @Override
     public void seeDetails(View sharedView, DashboardProgramModel dashboardProgramModel) {
-        Intent intent = new Intent(view.getContext(), TeiDataDetailActivity.class);
-        Bundle extras = new Bundle();
-        extras.putString("TEI_UID", teiUid);
-        extras.putString("PROGRAM_UID", programUid);
-        if (dashboardProgramModel.getCurrentEnrollment() != null)
-            extras.putString("ENROLLMENT_UID", dashboardProgramModel.getCurrentEnrollment().uid());
-        intent.putExtras(extras);
         ActivityOptionsCompat options = ActivityOptionsCompat.makeSceneTransitionAnimation(view.getAbstractActivity(), sharedView, "user_info");
-//        view.seeDetails(intent, options.toBundle());
-
         view.seeDetails(EnrollmentActivity.Companion.getIntent(view.getContext(),
                 dashboardProgramModel.getCurrentEnrollment().uid(),
                 dashboardProgramModel.getCurrentProgram().uid(),
```

Comment:
We should look for a way to move this calls (Intent and ActivityOptionsCompat) to the view. As well as removing View parameter from the method